docs/layouts/shortcodes/generated/expert_rest_section.html

```diff
@@ -86,6 +86,30 @@
             <td>Long</td>
             <td>The maximum time in ms for a connection to stay idle before failing.</td>
         </tr>
+        <tr>
+            <td><h5>rest.profiling.dir</h5></td>
+            <td style="word-wrap: break-word;">"/var/folders/19/05mc16fs0p13z3sj2y0qkzx00000ks/T/"</td>
```

Comment:
Thanks for the reminder. I think it will be right after adding `@Documentation.OverrideDefault("System.getProperty(\"java.io.tmpdir\")")` in `RestOptions#PROFILING_RESULT_DIR`